test/unit/news_article_test.rb

```diff
@@ -82,4 +83,30 @@ class NewsArticleTest < ActiveSupport::TestCase
   test "is not translatable when non-English" do
     refute build(:news_article, primary_locale: :es).translatable?
   end
+
+  test "can associate news articles with worldwide organisations" do
+    news_article = create(:news_article)
+    assert news_article.can_be_associated_with_worldwide_organisations?
+    assert news_article.worldwide_organisations.create(name: "Zimbabwean Embassy")
+  end
+
+  test "should be invalid when associating a worldwide organisation to a non-World-news-story news article type" do
+    non_world_news_story_news_article_types = [
+      NewsArticleType::NewsStory,
+      NewsArticleType::PressRelease,
+      NewsArticleType::GovernmentResponse,
+    ]
+
+    non_world_news_story_news_article_types.each do |news_article_type|
+      news_article = build(:news_article, news_article_type: news_article_type)
+      news_article.worldwide_organisations.build(name: "Zimbabwean Embassy")
+      refute news_article.valid?
```

Comment:
I usually try to check that the errors are on the bit we're testing in these type of tests too. You can get false passes/failures if the object is invalid for a different reason than we are testing.